pkg/reconciler/v1alpha1/processor/resources/deployment.go

```diff
@@ -91,3 +75,36 @@ func MakeDeployment(proc *streamv1alpha1.Processor) (*appsv1.Deployment, error)
 
 	return deployment, nil
 }
+
+func computeEnvironmentVariables(processor *streamv1alpha1.Processor) []corev1.EnvVar {
+	return append([]corev1.EnvVar{
+		{
+			Name:  "INPUTS",
+			Value: strings.Join(processor.Status.InputAddresses, ","),
+		},
+		{
+			Name:  "OUTPUTS",
+			Value: strings.Join(processor.Status.OutputAddresses, ","),
+		},
+		{
+			Name:  "GROUP",
+			Value: processor.Name,
+		},
+		{
+			Name:  "FUNCTION",
+			Value: "localhost:8080",
+		},
+	}, exposeOutputContentTypes("OUTPUT_CONTENT_", processor.Status.OutputContentTypes)...)
+}
+
+func exposeOutputContentTypes(keyPrefix string, outputContentTypes []string) []corev1.EnvVar {
```

Comment:
If you wonder why I go to great length to expose each content type separately, that's because it seems there is no reliable way to list content-type in a single envvar (no delimiter seems safe): https://www.w3.org/Protocols/rfc1341/4_Content-Type.html